lib/routes.js

```diff
@@ -0,0 +1,22 @@
+var router = require('express').Router(); // instantiate express router - new in Express 4.0 ^
```

Comment:
Yep, ideally I wanted to avoid using this here also and yes there would definitely be a lot of repetition when multiple route files are present. Wasn't sure of a best approach in this instance, I was going to try adding a function in here and then invoking it from server.js (and passing the Router instance as a parameter) but not sure if that works?